packages/@okta/i18n/src/properties/login.properties

```diff
@@ -597,8 +597,8 @@ primaryauth.submit = Sign In
 # {0} is the Okta account URL
 primaryauth.newUser.tooltip=This is the first time you are connecting to {0} from this browser
 primaryauth.newUser.tooltip.close=Close
-oktaVerify.description = To access this resource, your organization requires you to sign in using your device.
-oktaVerify.deviceOnlyDescription = To access {0}, your organization requires you to sign in with Okta Fastpass.
+oktaVerify.description = To access this resource, your organization requires you to sign in with Okta Fastpass.
```

Comment:
"oktaVerify.description" is only used in SignInWithDeviceOption.js. So updating the label.

Reply:
the `P` should be capitalized in `FastPass` to match our marketed term for it